pkg/operator/bootstrap.go

```diff
@@ -108,8 +108,8 @@ func (c *Controller) createPVC(accessMode corev1.PersistentVolumeAccessMode, cla
 		return err
 	}
 
-	// "standard" is the default StorageClass name, that was provisioned by the cloud provider
-	storageClassName := "standard"
+	// "standard-csi" is the default StorageClass name, that was provisioned by the cloud provider
```

Comment:
very minor nit: I would mention that `standard-csi` is for 4.11 and newer

Reply:
done